website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,545 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.1 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.1, which focused on fixing
+issues with the recently released Scala 3 support, Ammonite support and a couple
+of new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">261</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">96</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">16</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">5</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/24?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Ammonite support
+- New project provider
+- Worksheet dependency support
+- Multiline string improvements
+- Scala 2.13.3, 0.24.0, 0.25.0-RC1 support
+- Multiple workspace build tools selection
```

Comment:
Super small nitpick, but we want to make sure not to confuse people into thinking this is "multiple workspace" support in any way, so I'd recommend switching this up a bit.
```suggestion
- Multiple build tool selection within a workspace
```

Reply:
Fixed!